disvae/discriminator.py

```diff
@@ -0,0 +1,59 @@
+import torch
+from torch import nn
+
+
+class Discriminator(nn.Module):
+    def __init__(self,
+                 latent_dim=10,
+                 device=torch.device("cpu")):
+        r"""Discriminator proposed in [1].
```

Comment:
Does this need the string literal 'r' before the Docstring?

Reply:
I don't use any backslashes in the docstring so the 'r' is not necessary.  Happy to remove it.

Reply:
I don't feel strongly about it at all, I was more interested if you had a reason for doing it.